direct/src/distributed/AstronClientRepository.py

```diff
@@ -1,10 +1,10 @@
 """AstronClientRepository module: contains the AstronClientRepository class"""
 
+from pandac.PandaModules import STUint16, STUint32
```

Comment:
Perhaps this should be `from panda3d.direct import STUint16, STUint32` instead?

Reply:
Thanks! Wasn't sure which namespace to pull in for those.